src/app/board/board.component.html

```diff
@@ -1,8 +1,8 @@
 <app-header></app-header>
-<div class="container-fluid container-cards-pf">
+<div id="board_topWorkItems" class="container-fluid container-cards-pf">
   <h3>Top WorkItems</h3>
```

Comment:
Do we need IDs for h1, h2, h3, etc. During testing, do we check if a page has a title?